lti_consumer/tests/unit/test_lti.py

```diff
@@ -176,16 +176,39 @@ def test_get_signed_lti_parameters(self):
             'oauth_version': 'fake_version',
             'oauth_signature_method': 'fake_method',
             'oauth_consumer_key': 'fake_consumer_key',
-            'oauth_signature': u'fake_signature'
+            'oauth_signature': u'fake_signature',
+            u'custom_student_course_mode': 'verified',
+            u'custom_student_verification_status': 'verify_need_to_verify',
         }
         self.lti_consumer.xblock.has_score = True
         self.lti_consumer.xblock.ask_to_send_username = True
         self.lti_consumer.xblock.ask_to_send_email = True
+        self.lti_consumer.xblock.transmit_course_mode_and_status = True
```

Comment:
What about other cases in the code like:
- transmit_course_mode_and_status = False
- services unavailable
- Other return values (e.g. get_course_verification_status => None)
It seems that at a minimum, covering the first case can be done in a test with all these params set to false and ensuring that none of the optional params appear.